saleor/graphql/discount/mutations.py

```diff
@@ -486,6 +506,11 @@ class SaleInput(graphene.InputObjectType):
     products = graphene.List(
         graphene.ID, description="Products related to the discount.", name="products"
     )
+    variants = graphene.List(
+        graphene.ID,
+        descriptions="Product variant related to the discount.",
```

Comment:
```suggestion
        descriptions="{ADDED_IN_31} Product variant related to the discount.",
```

Reply:
Done (I didnt commit suggestion, because of lack of f-string literal.)